channels/applicationinsights-channel-js/src/applicationinsights-channel-js.ts

```diff
@@ -1 +1,2 @@
 export { Sender } from "./Sender";
+export { ISenderConfig } from "./Interfaces";
```

Comment:
We may also need to export from the 2 SKU's (main AISKU and the AISKULight) based on Logans' comment about using the basic SKU